css/main.css

```diff
@@ -0,0 +1,303 @@
+/*hamburger script*/
+
+* {
+    zoom: 1;
+    /*    behavior: url(/bower_components/css3pie/PIE.htc);*/
+}
+
+body {
+    font-family: 'Open Sans', sans-serif;
+    overflow-x: hidden;
+    width: 100%;
+    font-size: 16px;
+    background: #EAE2D7;
+    /* For browsers that do not support gradients */
+    background: -webkit-radial-gradient(white, #EAE2D7));
+    /* Safari 5.1 to 6.0 */
+    background: -o-radial-gradient(white, #EAE2D7));
+    /* For Opera 11.6 to 12.0 */
+    background: -moz-radial-gradient(white, #EAE2D7));
+    /* For Firefox 3.6 to 15 */
+    background: radial-gradient(white, #EAE2D7);
+    /* Standard syntax */
+    /*    behavior: url(/bower_components/css3pie/PIE.htc);*/
+}
+
+#main-header {
+    position: fixed;
+    top: 0px;
+    width: 100%;
+    background: #FFFFFF;
+    height: 70px;
+    line-height: 70px;
+}
+
+#main-header button {
+    margin: 10px;
+    color: black;
+    box-sizing: border-box;
+    border: 1px solid #CEB9B9;
+    border-radius: 5px;
+    background: none;
+    position: absolute;
+    top: 0;
+    right: 0;
+    padding: 0px 15px;
+    font-weight: bold;
+    cursor: pointer;
+    outline: none;
+    z-index: 1;
+    height: 50px;
+    /*    behavior: url(/bower_components/css3pie/PIE.htc);*/
+}
+
+.hamburger {
+    line-height: 45px;
+    font-size: 1.4em;
+}
+
+.cross {
+    width: 57px;
+    font-size: 3em;
+    line-height: 70px;
+}
+
+.menu {
+    top: 70px;
+    padding-left: 0px;
+    margin-top: 0px;
+    z-index: 1000000;
+    font-weight: bold;
+    font-size: 0.8em;
+    width: 100%;
+    background: #131313;
+    position: fixed;
+    text-align: center;
+}
+
+.menu ul {
+    margin: 0;
+    padding: 0!important;
+    list-style-type: none;
+    list-style-image: none;
+}
+
+.menu li {
+    display: block;
+    padding: 15px 0 15px 0;
+}
+
+.menu ul li a {
+    text-decoration: none;
+    text-align: center;
+    margin: 0px;
+    color: #fff;
+}
+
+.menu ul li a:hover {
+    color: #fff;
+    text-decoration: none;
+}
+
+.menu a {
+    text-decoration: none;
+    color: white;
+}
+
+.menu a:hover {
+    text-decoration: none;
+    color: white;
+}
+
+.cross {
+    display: none;
+}
+
+
+/*************** body elemnt ***************/
+
+.container {
+    margin-top: 160px;
```

Comment:
Let it be `.main-container` or something. .container is a bit confusing name.

Apart of that it should not have this kind of margin. Don't fix your header to avoid it.